IntegrationExample/Processing/View/ProcessingViewModel.swift

```diff
@@ -12,7 +12,6 @@ import MimiCoreKit
 
 final class ProcessingViewModel: ObservableObject {
     
-    var headphoneConnectivity: PartnerHeadphoneConnectivityController
```

Comment:
Is there a reason why `headphoneConnectivity` should be private here? Because of this we now need a wrapper method `simulateHeadphoneConnection ` (like I initially had it in AppDelegate) and another subscriber to connectivity state.